### PR TITLE
WIP: Translation of `Learn/Forms/Form_validation` page.

### PR DESCRIPTION
**Pull request by mouradev**
I started the translation of this content. The page was mixed with translated and no-translated content.

### CODE REVIEW

**Comment by mouradev** [comment]
```diff
diff --git a/files/pt-br/learn/forms/form_validation/index.html b/files/pt-br/learn/forms/form_validation/index.html
--- files/pt-br/learn/forms/form_validation/index.html
+++ files/pt-br/learn/forms/form_validation/index.html
@@ -59,13 +59,13 @@
 </ul>
 
 <p>No mundo real, desenvolvedores costumam usar validação de dados tanto no cliente como no servidor (em conjunto).</p>
 
-<h2 id="Using_built-in_form_validation">Usando validação HTML5 nativa</h2>
+<h2 id="Usando_validação_HTML5_nativa">Usando validação HTML5 nativa</h2>
 
-<p>One of the features of <a href="/en-US/docs/HTML/HTML5">HTML5</a> is the ability to validate most user data without relying on scripts. This is done by using <a href="/en-US/docs/Web/Guide/HTML/HTML5/Constraint_validation#Validation-related_attributes">validation attributes</a> on form elements, which allow you to specify rules for a form input like whether a value needs to be filled in, the minimum and maximum length of the data, whether it needs to be a number, an email address, or something else, and a pattern that it must match. If the entered data follows all the specified rules, it is considered valid; if not, it is considered invalid.</p>
+<p>Um dos recursos do <a href="/pt-BR/docs/HTML/HTML5">HTML5</a> é a habilidade de validar dados sem precisar de scripts. Isto é feito usando <a href="/pt-BR/docs/Web/Guide/HTML/HTML5/Constraint_validation#Validation-related_attributes">atributos de validação</a> em elementos de formulários, os quais permitem utilizar regras para os campos, como por exemplo, garantir que um valor será preenchido, especificar os valores mínimos e máximos, se precisa ser um número, endereço de e-mail, telefone, ou outra coisa, e até se um padrão de dados é utilizado. Se a entrada de dados respeitar todas as regras especificadas, é considerada válida; Se não, é considerada inválida.</p>
 
-<p>When an element is valid:</p>
+<p>Quando um elemento é válido:</p>
 
 <ul>
  <li>The element matches the {{cssxref(":valid")}} CSS pseudo-class; this will let you apply a specific style to valid elements.</li>
  <li>If the user tries to send the data, the browser will submit the form, provided there is nothing else stopping it from doing so (e.g., JavaScript).</li>

```

File: files/pt-br/learn/forms/form_validation/index.html [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+title: Validação de formulários
 slug: Learn/Forms/Form_validation
 translation_of: Learn/Forms/Form_validation
 original_slug: Web/Guide/HTML/Forms/Form_validation
 ---
 <div>{{LearnSidebar}}{{PreviousMenuNext("Learn/HTML/Forms/Sending_and_retrieving_form_data", "Learn/HTML/Forms/How_to_build_custom_form_widgets", "Learn/HTML/Forms")}}</div>
 
-<p class="summary">A validação de formulário nos ajuda a garantir que os usuários preencham os formulários no formato correto, garantindo que os dados enviados funcionem com êxito em nossos aplicativos. Este artigo apresentará conceitos e exemplos básicos sobre validação de formulário. Para mais informações adicionais, consulte o <a href="/en-US/docs/Web/Guide/HTML/HTML5/Constraint_validation">Constraint validation guide</a>.</p>
+<p class="summary">A validação de formulários nos ajuda a garantir que os usuários preencham os campos no formato correto, garantindo que os dados enviados funcionem em nossos aplicativos. Este artigo apresentará conceitos e exemplos básicos sobre validação de formulários. Para mais informações, consulte o <a href="/en-US/docs/Web/Guide/HTML/HTML5/Constraint_validation">Constraint validation guide</a>.</p>
```
Comment (NabeeL311) [inline files/pt-br/learn/forms/form_validation/index.html:9]: 1124#

**Comment by SphinxKnight** [comment]
@mouradev  / @mdn/yari-content-pt-br this PR is almost a year old and is based on a previous HTML version (now it needs to be in Markdown), what could we do to help finish this one?

**Comment by josielrocha** [comment]
Obrigado pela contribuição @mouradev!

Esse conteúdo foi migrado para Markdown..
Aproveitei parte do seu trabalho nesse PR (https://github.com/mdn/translated-content/pull/9227)

🚀

**Pull request closed without merging**